### PR TITLE
chore(refs T19787, T19759): Rebuild template for DpContextualHelp

### PR DESCRIPTION
**Pull request by salisdemos**
unFunctional Component and Move Template from Render to Template

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## UNRELEASED
 
+### Refactor
```
Comment (hwiem) [inline CHANGELOG.md:5]: I think this should be in the "Changed" category (if we want to follow [common changelog](https://common-changelog.org/) conventions), or, if we want to introduce a separate category for refactorings, it should be "Refactored"

Comment (salisdemos) [reply CHANGELOG.md:5]: Thanks. I never find the resource back. Will change that

**Review by hwiem — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 ## UNRELEASED
 
 ### Changed
+- ([#203](https://github.com/demos-europe/demosplan-ui/pull/203)) Move from Functional Component to Template in DpContextualHelp ([@salisdemos](https://github.com/salisdemos))
```
Comment (spiess-demos) [inline CHANGELOG.md:6]: I'm unsure if we have to mention internal implementation changes like those in the changelog. As i understand it, a changelog should communicate changed behavior, to enable consumers adapt their usages.

Comment (salisdemos) [reply CHANGELOG.md:6]: Me neither. I thought its worth mentioning it, so one can find errors faster if something strange happens after the update

**Pull request merged**